### ISSUE TITLE
Imperium - Adeptus Ministorum: Multiple Typos

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/wh40k#2623]
**File:** Imperium - Adeptus Ministorum.catz

**BattleScribe version:** 2.01.09

**Platform:** Windows

**Dropbox:** No

**Description:** 

- [x] Multiple spelling errors in Adeptus Ministorum file. 
- [x] War Hymn missing keyword INFANTRY. 
- [x] Penitent Engine spelling Error
- [x] Dialogus  spelling error

**Comment by CrusherJoe** [comment]
Can you be more specific on the "multiple spelling errors"? Or is the list you made exhaustive?

**Comment by jgholden** [comment]
The word Penitent was misspelled in several of the lists. Dialogus was also misspelled.  There were several other errors, such as referencing a Codex: Adepta Sororitas, which has not been released yet. Some other errors were simply words run together, and I corrected the spacing. I've updated the local file for the Ministorum, but am not sure how to upload it for review. The main error that was causing game issues was the Ministorum Priest's War Hymn ability. It was missing the INFANTRY keyword, and I was using the extra attack with the Penitent Engines. I was playing it incorrectly, and it should not have been used with the Penitent Engine as it is a Vehicle.
